src/include/log.h

```diff
@@ -6,6 +6,27 @@
  * See the file LICENSE for redistribution information.
  */
 
+/*
+ * WT_LSN --
+ *	A log sequence number, representing a position in the transaction log.
+ */
+struct __wt_lsn {
+	union {
+		struct {
+#ifdef	WORDS_BIGENDIAN
+			uint32_t file;
+			uint32_t offset;
+#else
+			uint32_t offset;
+			uint32_t file;
+#endif
+		} l;
+		uint64_t file_offset;
+	} u;
+};
+#define	lsn_file	u.l.file
+#define	lsn_offset	u.l.offset
```

Comment:
@sueloverso, not a big deal, but if you wanted to get rid of these macros, you could have:

```c
union __wt_lsn {
	struct {
#ifdef	WORDS_BIGENDIAN
		uint32_t file;
		uint32_t offset;
#else
		uint32_t offset;
		uint32_t file;
#endif
	} l;
	uint64_t file_offset;
};
```

Then `s/lsn_file/l.file/g` and `s/lsn_offset/l.offset/g`.  I don't feel strongly: I'm not a fan of macros to access struct/union fields, but we already have a bunch.